### DISCUSSION TITLE
sink downtime , buffering

### DISCUSSION THREAD

**Discussion by tomer-epstein** [vectordotdev/vector#7879] [Q&A] [answered]
Hi, 

We are interested how vector behave in case of sink downtime, does it support buffering .
Are there any documentation on this topic?

Thanks,
Tomer

**Comment by spencergilbert** [comment, marked as answer]
Hi! Sinks that support buffering will have a configuration section like such: https://vector.dev/docs/reference/configuration/sinks/elasticsearch/#buffer

Today we provide an in-memory or on-disk buffer depending on your needs
